setup.py

```diff
@@ -52,6 +52,7 @@
         'html-linter',
         'template-remover',
         'docutils',
+        'pathlib2',
```

Comment:
If the import is conditional, then the requirement should also be conditional, like the futures backport below.

However, see my comment above.